nimbus/src/lib.rs

```diff
@@ -162,9 +165,22 @@ pub struct EnrolledExperiment {
     pub branch_slug: String,
 }
 
+/// This is the currently supported schema version.
+pub const SCHEMA_VERSION: Version = Version {
+    major: 1,
+    minor: 0,
+    patch: 0,
+    pre: vec![],
+    build: vec![],
+};
+// XXX: In the future it would be nice if this lived in its own versioned crate so that
+// the schema could be decoupled from the sdk so that it can be iterated on while the
+// sdk depends on a particular version of the schema through the cargo.toml.
```

Comment:
I guess it depends on how the versions are going to be used in practice. I'd expect that a version bump would require corresponding changes to the rust code in the SDK and this version string ensures they are in sync?

Reply:
Yes, my thought was that this represents the version of the schema that is currently implemented here.  My biggest concern now is with backwards compatibility...  will we need multiple versions of the structs that represent the schema in order to support 2 versions backwards (per the requirement I learned from Kate), and how does that affect our persistent storage, etc.  

I'm in the process of writing up a short doc to define my understanding of what is being asked for with backwards compatibility, so I hope to be able to answer some of the questions I have.